connector/tests/src/test/java/org/oaebudt/edc/ManagementApiTransferTest.java

```diff
@@ -174,6 +201,43 @@ public void shouldSupportPullTransfer() throws IOException {
 
     }
 
+    @Test
+    public void shouldGetContractOfferViaFederatedCatalog() {
+        PROVIDER_FC.setAuthorizationToken(KEYCLOAK_EXTENSION.getToken());
+
+        final Map<String, Object> dataAddressProperties = Map.of(
+                "type", "HttpData-PULL",
+                "baseUrl", "http://localhost:8080/source"
+        );
+        final String assetId = createProviderAsset(PROVIDER_FC, dataAddressProperties);
+
+        final JsonObject requestBody = Json.createObjectBuilder()
+                .add("@context", Json.createObjectBuilder()
+                        .add("@vocab", "https://w3id.org/edc/v0.0.1/ns/"))
+                .add("@type", "QuerySpec")
+                .build();
+
+        await()
+                .atMost(Duration.ofSeconds(200))
```

Comment:
200 looks quite a lot :) is this really needed or is there a way to avoid the need to set it at all and make it work without having to set it? (e.g. by configuring the crawler accordingly)

Reply:
Well, I'm not sure. The crawler has to crawl the Target nodes periodically.
We need to wait for a certain period before it starts.
We can just reduce the time. 
60 seconds is okay actually. After a minute, it can be considered to have failed

Reply:
just because have to eventually wait a minute for a test to fail looks to be wasted time.
could be the crawler initial delay be set to 10 milliseconds? in that case we won't need to have such timeout at all and use the default (that's 10 seconds).
These would be 50 seconds saved for every failing execution :)

Reply:
Done.
Thanks @ndr-brt